pkg/alertmanager/amcfg.go

```diff
@@ -1718,6 +1724,22 @@ func (cb *ConfigBuilder) convertProxyConfig(ctx context.Context, in monitoringv1
 	return out, nil
 }
 
+func (cb *ConfigBuilder) convertGlobalJiraConfig(out *globalConfig, in monitoringv1.GlobalJiraConfig) error {
+	if in.APIURL != nil {
+		apiURLAllowed := cb.amVersion.GTE(semver.MustParse("0.28.0"))
```

Comment:
(suggestion) we can check version at the beginning because Jira integration requires at least 0.28.0 anyway.